konfera/utils.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+
+
+def collect_view_data():
```

Comment:
Instead of creating a util and invoking it in every single view. I'd create a [context processor](https://docs.djangoproject.com/en/1.10/ref/templates/api/#writing-your-own-context-processors)